src/clojure_elastic_apm/core.clj

```diff
@@ -83,6 +83,22 @@
        (set-label span k v))
      span)))
 
+(defn create-exit-span [{parent :parent
+                         span-name :name
+                         labels :labels
+                         type :type
+                         subtype :subtype
+                         action :action}]
+  (let [span (.startExitSpan (or parent (current-apm-span))
```

Comment:
Meillä on areena-vms-apissa lähtevät http-pyynnöt kääritty exit spaniin näin:
```clojure
(defn request [{:keys [url method] :as opts} & [callback]]
  (let [http-method  (-> method (name) (string/upper-case))
        apm-span (-> ^Span (apm/current-apm-span)
                     (.startExitSpan "ext" "http" http-method)
                     (.setServiceTarget "http" (url->host url))
                     (.setName (str http-method " " (without-api-auth url))))]
    (httpkit-client/request
     opts
     (fn [{:keys [error] :as response}]
       (when error (apm/capture-exception apm-span error))
       (apm/end apm-span)
       ((or callback identity) response)))))
```

jossa [`.setServiceTarget`](https://www.elastic.co/guide/en/apm/agent/java/current/public-api.html#api-span-set-service-target) on sellainen piirre, joka ei tällä with-apm-exit-span -toteutuksella ilmeisesti vielä onnistu.

Lähtevän http-pyynnön tapauksessa meillä siis asetetaan:
* type  = ext
* subtype = http
* service.target.type = http
* service.target.name = upstream hostin nimi, esimerkiksi `programs.api.yle.fi`

Sen ansiosta riippuvuudet nähdään nimettyinä APM:ssä:
![image](https://github.com/Yleisradio/clojure-elastic-apm/assets/880578/d564a981-32fb-4cce-906c-8dfebf0c9694)